src/sentry/workflow_engine/migration_helpers/issue_alert_migration.py

```diff
@@ -157,6 +158,11 @@ def _create_workflow_and_lookup(
         when_dcg = self._create_when_dcg(action_match=action_match)
         self._bulk_create_data_conditions(conditions=conditions, filters=filters, dcg=when_dcg)
 
+        enabled = True
+        rule_snooze = RuleSnooze.objects.filter(rule=self.rule, user_id=None).first()
+        if rule_snooze and rule_snooze.until is None:
```

Comment:
not bad to check but `until` is unused - I built the backend for it but we never built a front end